docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -1,168 +1,235 @@
+
+## Express Install
+
+With the firewall ports open you will now be able to source the script which provides the expressInstall function and
+execute it. You probably should consider DNS names before actually running the script. By default, the script will use the output of the
+`hostname -s` command to determine a reasonable default for the PKI that will be generated as well as the
+configuration files that are output.  The script allows you to configure these values and if you are deploying to a cloud
+provider (AWS, Azure, OCI, IBM, Digital Ocean, GCP, etc.) you will almost certainly want to use the **public** DNS name
+of your instance. You possibly  want to override the IP address to use as well, but using DNS names is superior to using
+IP addresses in case your IP changes.
+
+The quickest and easiest thing to do, is simply find your external DNS name and set it into the EXTERNAL_DNS environment
+variable. For example:
+
+```bash
+export EXTERNAL_DNS="acme.example.com"
+```
+
+:::note
+Make sure you have `jq` installed on your machine. From Debian distros that would look like:
+
+```bash
+sudo apt update && sudo apt install jq -y
+```
+
+:::
+
+Once you do that, you'll be able to
+execute these commands just as
+shown to have your
+controller and
+first edge router
+configured and ready to turn on:
+
+```bash
+# run me
+export EXTERNAL_IP="$(curl -s eth0.me)"       
+export ZITI_EDGE_CONTROLLER_IP_OVERRIDE="${EXTERNAL_IP}"
+export ZITI_EDGE_ROUTER_IP_OVERRIDE="${EXTERNAL_IP}"
+export ZITI_EDGE_CONTROLLER_HOSTNAME="${EXTERNAL_DNS}"
+export ZITI_EDGE_ROUTER_HOSTNAME="${EXTERNAL_DNS}"
+export ZITI_CTRL_PORT=8440
+export ZITI_EDGE_CONTROLLER_PORT=8441
+export ZITI_EDGE_ROUTER_PORT=8442
+```
+
+```bash
+# run expressInstall
+source /dev/stdin <<< "$(wget -qO- https://raw.githubusercontent.com/openziti/ziti/main/quickstart/docker/image/ziti-cli-functions.sh)"; expressInstall
+```
+
+### Systemd
+
+If the operating system you are deploying on supports it, after the commands above are run there will two other useful
+functions defined in your shell which will allow you to generate a systemd file for the controller and the edge router. This
+is useful to make sure the controller can restart automatically should you shutdown/restart the server. To generate these
+files run:
+
+```bash
+# run me
+createControllerSystemdFile
+createRouterSystemdFile "${ZITI_EDGE_ROUTER_RAWNAME}"
+```
+
+```bash
+# example output
+$ createControllerSystemdFile
+Controller systemd file written to: /home/ubuntu/.ziti/quickstart/ip-172-31-23-18/ip-172-31-23-18-edge-controller.service
+
+$ createRouterSystemdFile "${ZITI_EDGE_ROUTER_RAWNAME}"
+Router systemd file written to: /home/ubuntu/.ziti/quickstart/ip-172-31-23-18/ip-172-31-23-18-edge-router.service
+```
+
+After the files are generated, you can then install them for use by systemd by running:
+
+```bash
+# run me
```

Comment:
i never see comments like this on other code snippets fwiw. i think it's pretty clear that it's meant to be run. i don't see value in these "run me" comments